brand new story

```diff
@@ -0,0 +1 @@
+It was 5 am in the morning. John Doe was looking at his steaming cup of coffee and was thinking about the previous 72 hours of his life.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.